posts/zzz_DO_NOT_EDIT_introducing.../introducing_admiralneuro.qmd

```diff
@@ -0,0 +1,139 @@
+
+|                 |                            |                  |        |           |
+|---------------|---------------|---------------|---------------|---------------|
+| tracer          | pipeline                   | ref_region       | slope  | intercept |
+| 18F-Florbetapir | AVID FBP SUVR PIPELINE     | Whole Cerebellum | 183.07 | -177.26   |
+| 18F-Florbetaben | AVID FBB SUVR PIPELINE     | Whole Cerebellum | 156.06 | -148.13   |
+| 18F-Florbetapir | BERKELEY FBP SUVR PIPELINE | Whole Cerebellum | 188.22 | -189.16   |
+| 18F-Florbetaben | BERKELEY FBB SUVR PIPELINE | Whole Cerebellum | 157.15 | -151.87   |
+
+\
+Here is an example of the function in action:
+
+```{r, message=FALSE}
+# Using standard parameters
+library(admiral)
+library(admiralneuro)
+library(dplyr)
+
+adapet <- admiralneuro::admiralneuro_adapet %>%
+  filter(PARAMCD == "SUVRBFBB")
+
+adapet <- adapet %>%
+  admiral::derive_param_computed(
+    by_vars = c(
+      get_admiral_option("subject_keys"),
+      exprs(ADT, ADY, VISIT)
+    ),
+    parameters = c("SUVRBFBB"),
+    set_values_to = exprs(
+      AVAL = compute_centiloid(
+        tracer = "18F-Florbetaben",
+        pipeline = "BERKELEY FBB SUVR PIPELINE",
+        ref_region = "Whole Cerebellum",
+        suvr = AVAL
+      ),
+      PARAMCD = "CLBFBB",
+      PARAM = "Centiloid (CL) based on BERKELEY FBB",
+      AVALU = "CL"
+    )
+  )
+```
+
+```{r, eval=TRUE, echo=FALSE, message=FALSE}
+library(admiraldev)
+dataset_vignette(adapet %>% filter(PARAMCD == "CLBFBB") %>% arrange(USUBJID, ADT), display_vars = exprs(USUBJID, PARAMCD, PARAM, VISIT, AVAL, AVALU))
+```
+
+\
+Alternatively, the user can specify a custom slope and intercept to the function using the **custom_slope** and **custom_intercept** parameters.
+
+### Templates and Vignette
+
+The {admiralneuro} package website contains the [Creating ADTPET and ADAPET Vignette](https://pharmaverse.github.io/admiralneuro/articles/adpet.html) which guides users through the creation of both ADTPET (Tau PET) and ADAPET (Amyloid PET) ADaMs.
+This covers the use of the `compute_centiloid()` function above, as well as other therapeutic area-specific considerations for these BDS datasets such as derivation of criterion flag variables for Centiloid categories.
+
+The vignette accompanies the ADTPET and ADAPET template scripts, which can be used as a starting point for creating these ADaMs.
+These scripts can be loaded and saved directly from the R console by running either of the following:
+
+```{r, eval=FALSE}
+use_ad_template("ADTPET", package = "admiralneuro")
+```
+
+```{r, eval=FALSE}
+use_ad_template("ADAPET", package = "admiralneuro")
+```
+
+# Conclusion
+
+We hope that this initial release of {admiralneuro} will be useful to those working in the Alzheimer's disease space, and we look forward to expanding the package in future releases to broaden its scope and functionality within this and other neurological disease areas.
```

Comment:
@jwang-lilly do you want to be a bit more specific here and talk about some of the 0.2.0 features or would you rather keep it vague?

Reply:
We’re excited to share the first release of {admiralneuro}, and we hope it will be a valuable resource for researchers working in the Alzheimer’s disease space. Future updates will expand the package to include alpha-synuclein pathology, a hallmark of Parkinson’s disease that is also found in other neurodegenerative conditions, including Alzheimer’s disease. We’ll also be adding functions to explore olfactory impairment, a fascinating early signal connected to both Parkinson’s disease and Alzheimer’s disease.